File: Beecrowd/Beginner/2003 - 2176/2028 - Sequence of Sequence/C++/2028_SequenceofSequence.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {
    int lastValue, i = 1;

    while (cin >> lastValue) {
        int sum = 1;

        for (int j = 1; j <= lastValue; j++) sum += j;

        if (lastValue == 0) cout << "Caso " << i << ": " << sum << " numero" << endl;
        else cout << "Caso " << i << ": " << sum << " numeros" << endl;

        cout << 0;

        for (int j = 1; j <= lastValue; j++) {
            for (int k = 1; k <= j; k++) {
                cout << " " << j;
            }
        }

        cout << endl << endl;

        i++;
    }

    return 0;
}

File: Beecrowd/Beginner/2003 - 2176/2028 - Sequence of Sequence/Python/2028_SequenceofSequence.py
test_cases = 0
while True:
    try:
        test_cases += 1
        number = int(input())
        sequence = [0]
        
        for i in range(1, number+1):
            for j in range(i):
                sequence.append(i)
                
        if number == 0:
            print(f"Caso {test_cases}: {len(sequence)} numero")
        else:
            print(f"Caso {test_cases}: {len(sequence)} numeros")
            
        print(*sequence)
        print()
    except EOFError:
        break
